.github/workflows/ci.yml

```diff
@@ -54,11 +54,17 @@ jobs:
       uses: psf/black@stable
       with:
         src: "../test/${{ matrix.copier_config.foldername }}/src"
+    - name: Install notebook requirements
+      if: ${{ true }}
```

Comment:
Is this line necessary?

Reply:
It's the part I mention in the PR description above, where the `if: ${{ true }}` is a placeholder for conditioning this on whether or not the user has chosen the notebook option in the template. 

Strictly speaking it isn't necessary, but I plan to update it with actual logic as soon as I address issue #77.